report/lambda_function.py

```diff
@@ -293,7 +323,49 @@ def plot_table(df_releases: DataFrame, rows: int) -> Chart:
                 orient="top",
                 labelAngle=0,
                 title=None,
-                ticks=False
+                ticks=False,
+                labelFontSize=26,
+                labelLimit=400
+            ),
+            scale=alt.Scale(padding=10),
+            sort=None,
+        ),
+        alt.Y("index", type="ordinal", axis=None),
+        alt.Text("value", type="nominal"),
+    ).properties(
+        height=1000,
+        width=1300
+    )
+    return chart
+
+
+def plot_table_small(df_releases: DataFrame, rows: int) -> Chart:
+    """
+    Create a table from a given DataFrame
```

Comment:
This is a bit of a vague docstring: what table, what dataframe? - maybe note in the docstrings what columns the table will have?